src/commands/env/rename.ts

```diff
@@ -41,7 +41,7 @@ export default class EnvRenameCmd extends Command {
 		id: Args.integer({
 			required: true,
 			description: "variable's numeric database ID",
-			parse: async (input) => parseAsInteger(input, 'id'),
+			parse: (input) => Promise.resolve(parseAsInteger(input, 'id')),
```

Comment:
Do we need the Promise.resolve in all these cases? Does parse require a promise? I would assume we can just remove the async and not have a Promise.resolve at all